apps/src/lib/kits/maker/boards/microBit/MicroBitBoard.js

```diff
@@ -267,6 +267,18 @@ export default class MicroBitBoard extends EventEmitter {
       false /* shouldDestroyComponents */
     );
   }
+
+  calibrateCompass() {
```

Comment:
I notice we have 'calibrateCompass' and 'compassCalibration' - I might suggest we converge on a term (it could be different than the one we show the user on the button) to make it easier to talk and think through this code.